### PR TITLE
[MDS-5944] application submission success fail page

### PR DESCRIPTION
**Pull request by asinn134**
## Objective 

[MDS-5944](https://bcmines.atlassian.net/browse/MDS-5944)

- Added submission status page in common package to display if authorization jobs have succeeded or failed. The page's layout is determined by the ams_status_code of the authorizations, if any authorization has a "400" ams_status_code then we will display the fail page, else the success page will be displayed

Screenshot (363)

Screenshot (366)

- Made some styling changes based on suggestions by Denise

     1.  The input text stating a file upload field is required appears at the bottom of the file uploader and there is a bit of a gap between the text and the actual file uploader it is referring to. The change was to move the text closer to the uploader it refers to. 
Screenshot (364)

     2. Fixed a css issue in Major project's Purpose and Authorization section where the width would stretch depending on the length of the file's name being uploaded.

Screenshot (367)

### CODE REVIEW

File: services/common/src/components/projectSummary/ProjectSubmissionStatusPage.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+              </Col>
+            </Row>
+            <br />
+            <Typography.Paragraph>
+              <Typography.Title level={4}>Submission Failed.</Typography.Title>
+              Please retry submission in Project Description Overview in a few minutes.
+            </Typography.Paragraph>
+            <div>
+              <p>
+                <Link to={{}}>
```
Comment (asinn134) [inline services/common/src/components/projectSummary/ProjectSubmissionStatusPage.tsx:96]: This link is empty at the moment but will redirect to the project description overview page that Henry is working on.

File: services/common/src/components/projectSummary/ProjectSubmissionStatusPage.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+import { Row, Col, Typography, Button } from "antd";
+import CheckCircleOutlined from "@ant-design/icons/CheckCircleOutlined";
+import ArrowLeftOutlined from "@ant-design/icons/ArrowLeftOutlined";
+import CloseCircleOutlined from "@ant-design/icons/CloseCircleOutlined";
+import LinkButton from "../common/LinkButton";
+import { ENVIRONMENT } from "@mds/common/constants";
+import * as API from "@mds/common/constants/API";
+import { AMS_STATUS_CODE_FAIL } from "@mds/common/constants";
+
+export const ProjectSubmissionStatusPage = () => {
```
Comment (simensma-fresh) [inline services/common/src/components/projectSummary/ProjectSubmissionStatusPage.tsx:12]: Could we add in a snapshot test for this component?

**Review by simensma-fresh — CHANGES_REQUESTED** [review]

File: services/common/src/components/projectSummary/ProjectSubmissionStatusPage.tsx (hunk 2 above)
Comment (asinn134) [reply services/common/src/components/projectSummary/ProjectSubmissionStatusPage.tsx:12]: This has been added now.

**Review by simensma-fresh — APPROVED** [review]

File: services/common/src/setupTests.ts [hunk 3]
```diff
@@ -13,6 +13,13 @@ Enzyme.configure({ adapter: new Adapter() });
   setTimeout(callback, 0); // eslint-disable-line @typescript-eslint/no-implied-eval
 };
 
+(<any>global).GLOBAL_ROUTES = {
```
Comment (matbusby-fw) [inline services/common/src/setupTests.ts:16]: 👍🏻

**Review by matbusby-fw — APPROVED** [review]
Nice! Looks great!

**Pull request merged**